docs/user/science.rst

```diff
@@ -111,17 +111,17 @@ Here are some popular activities that are well-supported by Jupyter Book:
 * Data science software documentation
 
 
-:doc:`Visit the gallery of solutions built with Jupyter Book <executablebook:gallery>`
+:doc:`Visit the gallery of solutions built with Jupyter Book » <executablebook:gallery>`
 
 
 Ready to get started?
 """""""""""""""""""""
 
 .. Note that this is a deliberate repitition of a previous segment. Should it repeat? Maybe not, but for now it's nice to be sure that people see it.
 
-* All new to this? Take the official :external+jupyterbook:doc:`Jupyter Book Tutorial <start/your-first-book>`
-* Curious for practical code? See :doc:`/examples`
-* Familiar with Sphinx? See :doc:`/guides/jupyter`
+* All new to this? Take the official :external+jupyterbook:doc:`Jupyter Book Tutorial » <start/your-first-book>`
+* Curious for practical code? See the list of :doc:`Example Projects » </examples>`
+* Familiar with Sphinx? Read :doc:`How to use Jupyter notebooks in Sphinx » </guides/jupyter>`
```

Comment:
Why are you using a `»` at the end of the links? If it's to denote that it's an external link, I think there is some consensus about using the "square with the arrow". Like this one: https://fontawesome.com/icons/arrow-up-right-from-square?s=solid

Reply:
THANKS!! :heart: I was actually contemplating exactly what convention to use here. Will see if this can be used.